pom.xml

```diff
@@ -181,6 +182,23 @@
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.owasp</groupId>
+                <artifactId>dependency-check-maven</artifactId>
+                <version>${dependency-check-plugin.version}</version>
+                <configuration>
+                    <suppressionFiles>
+                        <suppressionFile>suppress.xml</suppressionFile>
```

Comment:
looks like we'll need to include a suppress.xml file - see martin's PR - https://github.com/companieshouse/auth-code-notification/pull/44/files

Reply:
Suppress added.

Reply:
looks good, just waiting for sonar checks to complete before approval